modules/optics/arrow-optics/src/main/kotlin/arrow/optics/Prism.kt

```diff
@@ -65,15 +65,6 @@ interface PPrism<S, T, A, B> : PPrismOf<S, T, A, B> {
       override fun reverseGet(b: B): T = reverseGet(b)
     }
 
-    /**
-     * Invoke operator overload to create a [PPrism] of type `S` with focus `A` with a [PartialFunction]
-     * Can also be used to construct [Prism]
-     */
-    operator fun <S, A> invoke(partialFunction: PartialFunction<S, A>, reverseGet: (A) -> S): Prism<S, A> = Prism(
```

Comment:
Same here

Reply:
@nomisRev how did you solve this?

Reply:
https://github.com/arrow-kt/arrow/pull/1419/files#diff-7e03fe423fc60d0d4ff339cb00e547c1R283